docs/content/tutorials/building-comfort/_index.md

```diff
@@ -0,0 +1,1161 @@
+
+This means, we can spend more time in focusing on how the dashboard UI will
+  look, rather than worrying about implementing and operating new backend
+  infrastructure.
+
+Following is how the architecture looks like with Drasi:
+
+```
+|--Existing System--|
+                              
+Sensors → PostgreSQL →→→→→→ Drasi Source → Drasi Queries → Drasi Reaction → Frontend
+                              (config)     (Declarative)       (config)
+|-----No Change-----|
+```
+
+## Building Comfort with Drasi
+
+The rest of this tutorial will guide us in setting up a mock building
+  management scenario and demonstrate how we can get a Reactive Dashboard built
+  for existing systems with just some config files and Cypher queries.
+
+### Tutorial Modes
+
+You can follow along the steps below in a Github codespace, a VSCode Dev
+  Container or your own Kubernetes environment.
+
+{{< tabpane >}}
+
+{{% tab header="Github Codespaces" text=true %}}
+
+The easiest way to follow along with this tutorial is to launch a Github
+  Codespace using the link below. This will allow you to run the example
+  application within your browser without setting up anything on your own
+  machines.
+
+[![Open in Github Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/drasi-project/learning?devcontainer_path=.devcontainer%2Fbuilding-comfort%2Fdevcontainer.json)
+
+This will open a page with some configuration options. Make sure that the
+  'Branch' selected is `main` and set the 'Dev Container configuration' to
+  'Building Comfort with Drasi'.
+
+{{< figure src="Screenshot_BuildingComfort_CodespacesConfig.png"
+  alt="Screenshot showing configuration for Codespaces" width="80%" >}}
+
+{{% /tab %}}
+
+{{% tab header="VS Code Container" text=true %}}
+
+To follow along with a Dev Container, you will need to install:
+- [Visual Studio Code](https://code.visualstudio.com/)
+    (or [Insiders Edition](https://code.visualstudio.com/insiders))
+- Visual Studio Code [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) 
+- [docker](https://www.docker.com/get-started/)
+
+Next, [clone the learning repo from Github](https://github.com/drasi-project/learning),
+  and open the repo in VS Code. Make sure that Docker deamon
+  (or Docker Desktop) is running.
+
+Once the solution is open in VS Code, follow these steps:
+- Press Cmd + Shift + P (on MacOS) or Ctrl + Shift + P (Windows or Linux) to
+    launch the command palette.
+- Select `Dev Containers: Rebuild and Reopen in Container`.
+- Select the `Building Comfort with Drasi` option to launch this tutorial.
+
+{{% /tab %}}
+
+{{% tab header="Local Setup" text=true %}}
+
+You need to have your own Kubernetes cluster setup.
+You can use any Kubernetes setup.
+For a local testing setup, you can choose one of alternatives
+  like Kind, Minikube or k3d.
+
+Make sure that `kubectl` on your system points to your Kubernetes cluster.
```

Comment:
We should also highlight the fact we need things like `drasi` and `npm` if the user chooses to use the local setup approach

Reply:
- There are instructions in tutorial for installing Drasi CLI for local setup.
- Added info for npm & python pre-requisites.